### PR TITLE
Gas schedule error when loading fails

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added new gas schedule + version
- added prints when the gas schedule fails to load
- fixed local scripts

### CODE REVIEW

File: vm/factory/systemSCFactory.go [hunk 1]
```diff
@@ -13,6 +14,8 @@ import (
 	"github.com/mitchellh/mapstructure"
 )
 
+var log = logger.GetOrCreate("core/forking")
```
Comment (sasurobert) [inline vm/factory/systemSCFactory.go:17]: Is this the correct package?

File: vm/factory/systemSCFactory.go [hunk 2]
```diff
@@ -148,6 +152,8 @@ func (scf *systemSCFactory) GasScheduleChange(gasSchedule map[string]map[string]
```
Comment (sasurobert) [inline vm/factory/systemSCFactory.go:152]: Log error on line 150 as well

**Review by sasurobert — CHANGES_REQUESTED** [review]

File: vm/factory/systemSCFactory.go (hunk 1 above)
Comment (iulianpascalau) [reply vm/factory/systemSCFactory.go:17]: :man_facepalming:

Comment (iulianpascalau) [reply vm/factory/systemSCFactory.go:17]: done

File: vm/factory/systemSCFactory.go (hunk 2 above)
Comment (iulianpascalau) [reply vm/factory/systemSCFactory.go:153]: done

**Review by sasurobert — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Review by valentincraciun99 — APPROVED** [review]
system testing passed

**Pull request merged**